bin/rector

```diff
@@ -1,5 +1,55 @@
 #!/usr/bin/env php
 <?php
```

Comment:
Could you also `strict_types()` here?

Reply:
I got a message when trying to do it:
`Fatal error: strict_types declaration must be the very first statement in the script` :cold_sweat:

Reply:
Did it. Sorry, I forgot :sweat_smile:

Reply:
That was probably the reason.
But it works here: https://github.com/phpstan/phpstan/blob/master/bin/phpstan

Have you managed to figure that out?
It might be related to include file

Reply:
Oh, if `PHPStan` has, we can have it too. Gonna search how to fix it :man_technologist:

Reply:
For some weird reason, it was related to my IDE, not to PHP itself. Gonna duplicate this to `Symplify/EasyCodingStandard`

Reply:
Thanks for investigation!